api/database/migrations/2023_04_24_195500_experience_evidence_setup.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Query\Expression;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('pool_candidate_experience', function (Blueprint $table) {
```

Comment:
Let's rename the table slightly to make it clear why these are being connected in this case. We could call it `pool_candidate_criteria_experience` or `pool_candidate_minimum_criteria_experience`?